dashboard/app/views/levels/editors/_applab.html.haml

```diff
@@ -111,3 +118,10 @@
 .field
   =f.label 'Disable code debugger (Console is still enabled)'
   = boolean_check_box f, :debugger_disabled
+
+.field.teacher-only-markdown
+  %p Edit Teacher Only Markdown:
```

Comment:
Possible the levelbuilder edit page should call out that this is visible only to _authorized_ teachers?